Changelog.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 2.2.4 - 2021-02-23
+- Added ```@azure/core-asynciterator-polyfill``` package as a dependency. Please refer [Issue #8764](https://github.com/Azure/azure-sdk-for-js/issues/8764) for further details.
```

Comment:
I don't feel like the linked issue actually adds much clarity around what importing this polyfill does for consumers

Reply:
Agreed, lets add a one liner here instead

Reply:
On second thoughts... do the packages that use ms-rest-js even have async iterators?

Reply:
Updated Changelog with an one liner. Could you please check if it is acceptable? @ramya-rao-a For the second question, If my memory serves correct, I found this issue only when working with one of the cognitive services which depends on ms-rest-js.

Reply:
> If my memory serves correct, I found this issue only when working with one of the cognitive services which depends on ms-rest-js.

Let's confirm that. From what I know, the v4 code gen does not use async iterators

Reply:
On further checking, I could not find any async iterators within cognitive services. It might be related to search sdk with v4 generator whose custom layer had async iterators as [here](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/search/search-documents/src/searchClient.ts#L341). But even then it was dependent only on core-http and not on ms-rest-js. So, the answer is Yes. I believe we do not have async iterators on this.  Any thoughts???

P.S: Now search documents SDK is moved to V6 SDK generators.

Reply:
I thought so too. So, in that case, we don;t need this change in ms-rest-js